karma.conf.js

```diff
@@ -1,3 +1,6 @@
+var isparta = require('isparta');
+var istanbul = require('browserify-istanbul');
```

Comment:
what a name `istanbul` :smile:. I think now we should create something like `browserify-kathmandu` :wink: